src/viur/toolkit/importer/importable.py

```diff
@@ -356,9 +357,10 @@ def do_import(
             ):
                 updated += 1
 
-                # if total >= 5:
-                #    skellist = ()
-                #    break
+                if debuglimit := import_conf.get("debuglimit"):
+                    if total >= debuglimit:
+                        skellist = ()
```

Comment:
```suggestion
                        skellist = []
```
This should fix the type checker error

> `src/viur/toolkit/importer/importable.py:362: error: Incompatible types in assignment (expression has type "tuple[()]", variable has type "list[Any]")  [assignment]`


https://github.com/viur-framework/viur-toolkit/actions/runs/12604258821/job/35130816799#step:6:14